Pre-DevIncubator/Models/Engine/ElectricalEngine.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Pre_DevIncubator.Models.Engine
+{
+    public class ElectricalEngine : AbstractEngine
+    {
+        public double ElectricityConsumption { get; init; }
+        public int HorsePowers { get; set; }
+
+        public ElectricalEngine(double electricityConsumption, int battaryCapacity):
+            base("Electrical", 0.1)
```

Comment:
Could you configure your IDE formatter correctly? Why is there not space before ':' ?